src/main/java/DayOfTheWeek.java

```diff
@@ -0,0 +1,41 @@
+public class DayOfTheWeek {
+    Day day;
+
+    public DayOfTheWeek(Day day) {
+        this.day = day;
+    }
+
+    public void  PrintDayName() {
+        switch (day) {
+            case Monday:
```

Comment:
czy tutaj faktycznie jest potrzebny tak długi case?
Rzuć okiem na to jakie metody ma enum. Może uda sie to zapisać.. o wiele krócej. :)